### ISSUE TITLE
Stepper and Stepper Item - Ensure active user interface components have sufficient contrast - (2036570233)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7835]
### Violation: 
 Ensure active user interface components have sufficient contrast 

### WCAG Reference: 
- [1.4.11 Non-test Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) 

### Severity: 
 6 

### Media Type: 
 Color and Contrast 

--- 
### Areas for Remediation: 
- Title: Stepper and Stepper Item - Ensure active user interface components have sufficient contrast - (2036570233) 
- Module: Stepper and Stepper Item 
**Issue**
Light gray form field borders on a background have a contrast ratio below 3.00:1, when checking with "Toggle theme" on.

Foreground: #5A5A5A
Background :#353535
Contrast ratio: 1.8:1

**User Impact**
Users with low vision will have difficulty identifying this content.

 
**Suggestion**
Ensure active user interface components have sufficient contrast. The required minimum contrast ratio is 3.00:1.

Common examples of qualifying components include text field borders, checkmarks for checkboxes, fillings for radio buttons, focus indicators, and icon-only controls.

For borders, the "adjacent color" can be the color that touches the outside of the border or the color that touches the inside of the border. Contrast with both is not required.

Disabled controls that cannot be navigated to with the keyboard are exempt from this requirement.

Refer to the Accessible Color Picker extension or Color Contrast Checker site for assistance: https://www.accessibility.dev/

 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Closing in favor of the duplicate https://github.com/Esri/calcite-design-system/issues/7822
